### DISCUSSION TITLE
`mtail` grammar online editor/tester

### DISCUSSION THREAD

**Discussion by mingodad** [google/mtail#769] [Show and tell]
I've just added `mtail` grammar to https://mingodad.github.io/parsertl-playground/playground/ a `Yacc/Lex` compatible online editor/tester (select `MTail parser` from `Examples` then click `Parse` to see a parser tree for the content in `Input source` editor).

I hope it can help debug/develop/document/test this project grammar !

Any feedback is welcome !

**Comment by jaqx0r** [comment]
Wow, cool!

Can you rename to `mtail`, all lowercase?

Also would you like to send a PR adding a link to your page to our README?

**Reply by mingodad** [reply]
I've changed `MTail` to `mtail`.
